velox/connectors/Connector.h

```diff
@@ -367,6 +367,9 @@ class ConnectorFactory {
 
   virtual ~ConnectorFactory() = default;
 
+  /// Initialize is called during the factory registration.
+  virtual void initialize() = 0;
```

Comment:
Could have an empty default implementation since it's not mandatory and more of a convenience to implement this method.

Reply:
done!